src/frontend/js/pages/DashboardOpenEdxProfile/index.tsx

```diff
@@ -0,0 +1,223 @@
+import { Input } from '@openfun/cunningham-react';
+import { FormattedMessage, defineMessages, useIntl } from 'react-intl';
+import { useSession } from 'contexts/SessionContext';
+import useOpenEdxProfile from 'hooks/useOpenEdxProfile';
+import { DashboardBox } from 'widgets/Dashboard/components/DashboardBox';
+import { DashboardCard } from 'widgets/Dashboard/components/DashboardCard';
+import { RouterButton } from 'widgets/Dashboard/components/RouterButton';
+import Form from 'components/Form';
+import context from 'utils/context';
+
+const messages = defineMessages({
+  sectionHeader: {
+    id: 'components.DashboardOpenEdxProfile.header',
+    description: 'Title of the dashboard open edx profile block',
+    defaultMessage: 'Profile',
+  },
+  baseInformationHeader: {
+    id: 'components.DashboardOpenEdxProfile.baseInformationHeader',
+    description: 'Title of the open edx profile form "basic information" block',
+    defaultMessage: 'Basic account information',
+  },
+  additionalInformationHeader: {
+    id: 'components.DashboardOpenEdxProfile.additionalInformationHeader',
+    description: 'Title of the open edx profile form "additional information" block',
+    defaultMessage: 'Additional account information',
+  },
+  editButtonLabel: {
+    id: 'components.DashboardOpenEdxProfile.EditButtonLabel',
+    description: 'Label of the edit button link of the open edx profile form',
+    defaultMessage: 'Edit your profile',
+  },
+
+  usernameInputLabel: {
+    id: 'components.DashboardOpenEdxProfile.usernameInputLabel',
+    description: 'Label of the openEdx profile "username" input',
+    defaultMessage: 'Username',
+  },
+  fullNameInputLabel: {
+    id: 'components.DashboardOpenEdxProfile.fullNameInputLabel',
+    description: 'Label of the openEdx profile "fullName" input',
+    defaultMessage: 'Full name',
+  },
+  emailInputLabel: {
+    id: 'components.DashboardOpenEdxProfile.emailInputLabel',
+    description: 'Label of the openEdx profile "email" input',
+    defaultMessage: 'Email',
+  },
+  languageInputLabel: {
+    id: 'components.DashboardOpenEdxProfile.languageInputLabel',
+    description: 'Label of the openEdx profile "langue" input',
+    defaultMessage: 'Langue',
+  },
+  countryInputLabel: {
+    id: 'components.DashboardOpenEdxProfile.countryInputLabel',
+    description: 'Label of the openEdx profile "country" input',
+    defaultMessage: 'Country',
+  },
+
```

Comment:
ditto

Reply:
i've added space to create tree groups:
* ux elements (button, title, etc)
* first form label
* second form label

I realized there that i forgot a space to separate the descriptions.